static/js/main.js

```diff
@@ -398,7 +398,11 @@ function takeScreenshot(source, x, y, width, height, format = 'image/jpeg') {
     return canvas.toDataURL(format);
 }
 
-function deleteScreenshot(screenshot_id) {
+function deleteScreenshot(screenshot_id=null) {
+    // If no id is given, delete most recent screenshot.
+    if (screenshot_id == null) {
+        screenshot_id = document.querySelector('.screenshot-area .screenshot').getAttribute('data-id');
+    }
```

Comment:
Missing space after this if.
We generally don't want executing code directly after a conditional (without vertical whitespace).